api/organisations/tests/test_views.py

```diff
@@ -582,6 +583,45 @@ def setUp(self) -> None:
         )
         self.subscription = Subscription.objects.get(organisation=self.organisation)
 
+    @mock.patch("organisations.views.extract_subscription_metadata")
```

Comment:
I'm not happy about having to continue to use these class based tests but ripping these up into functional pytest tests would be a fairly large effort that I don't think this PR warrants right now.